olive/engine/engine.py

```diff
@@ -803,21 +802,15 @@ def _run_passes(
         output_model_hash = None
 
         if cloud_cache_config.enable_cloud_cache:
-            if not (
-                model_config.type.lower() == "hfmodel"
-                and create_resource_path(model_config.config.get("model_path")).is_string_name()
-            ):
+            if not is_valid_cloud_cache_model(model_config):
                 logger.warning(
-                    "Only HfModel with huggingface id as model_path is supported by cloud cache. Setting"
-                    " enable_cloud_cache=False."
+                    "Only HfModel with huggingface id as model_path "
+                    "or HFModel from Azure ML registry model is supported by cloud cache. "
```

Comment:
- HfModel vs HFModel
- HFModel is Olive concept that user may need to be familiar with.

Reply:
the inconsistency between `HfModel` and `HFModel` in this docstring still exists. My intention was to use `HfModel` as the casing but I missed updating here https://github.com/microsoft/Olive/blob/9e2c817b126f38d462680d72b20def711b26c668/olive/model/handler/hf.py#L27

Reply:
Will update to HfModel.